### PR TITLE
NEXUS-25052: P2 format. Administrative

### PR DESCRIPTION
**Pull request by proteus1121**
ticket: https://issues.sonatype.org/browse/NEXUS-25052

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,6 +15,12 @@
 # Nexus Repository Manger p2 Format
 [![Maven Central](https://img.shields.io/maven-central/v/org.sonatype.nexus.plugins/nexus-repository-p2.svg?label=Maven%20Central)](https://search.maven.org/search?q=g:%22org.sonatype.nexus.plugins%22%20AND%20a:%22nexus-repository-p2%22) [![CircleCI](https://circleci.com/gh/sonatype-nexus-community/nexus-repository-p2.svg?style=shield)](https://circleci.com/gh/sonatype-nexus-community/nexus-repository-p2) [![Join the chat at https://gitter.im/sonatype/nexus-developers](https://badges.gitter.im/sonatype/nexus-developers.svg)](https://gitter.im/sonatype/nexus-developers?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge) [![DepShield Badge](https://depshield.sonatype.org/badges/sonatype-nexus-community/nexus-repository-p2/depshield.svg)](https://depshield.github.io)
 
+> **Huzzah!** p2 is now part of Nexus Repository Manager. Version 3.20 includes the p2 plugin by default. 
+>The plugin source code is now in [nexus-public](https://github.com/sonatype/nexus-public) in [nexus-repository-p2](https://github.com/sonatype/nexus-public/tree/master/plugins/nexus-repository-p2).
```
Comment (joedragons) [inline README.md:19]: The second link (nexus-repository-p2) 404s for me.  Is it TBD or is there something wrong (perms, mislink)?

File: README.md [hunk 2]
```diff
@@ -15,6 +15,12 @@
...
+> **Filing issues:** Upgrade to the latest version of Nexus Repository Manager 3, to get the latest fixes and improvements, before filing any issues or feature requests at https://issues.sonatype.org/.
+> **Upgrading:** If you are using a version prior to 3.20 and upgrade to a newer version you will not be able to install the community plugin. 
+>No other changes are required, and your existing data will remain intact.
```
Comment (joedragons) [inline README.md:22]:
```suggestion
>No other changes are required and your existing data will remain intact.
```
Rationale: This isn't an oxford comma with 2 items, it's just a redundant conjunction (trivial).

**Comment by joedragons** [comment]
Q: Is the intention each bold section is on a new line or it be one block?  Currently it's one block, so I figured I'd check.

File: README.md (hunk 1 above)
Comment (proteus1121) [reply README.md:19]: @joedragons it will be available after merging https://github.com/sonatype/nexus-internal/pull/5872

**Review by joedragons — APPROVED** [review]
If you wanted the new section not all one block (see my Q), I think you need some more line breaks.  That's trivial tho.  Approving.

**Comment by proteus1121** [comment]
@joedragons Yes, I missed that it's one block, should be line breaks definitely, thanks. https://github.com/sonatype-nexus-community/nexus-repository-p2/pull/79/commits/0e430761c04f653678f2df3b948656fb97a50360

**Review by sergeyonofreychuck — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -15,6 +15,14 @@
 # Nexus Repository Manger p2 Format
 [![Maven Central](https://img.shields.io/maven-central/v/org.sonatype.nexus.plugins/nexus-repository-p2.svg?label=Maven%20Central)](https://search.maven.org/search?q=g:%22org.sonatype.nexus.plugins%22%20AND%20a:%22nexus-repository-p2%22) [![CircleCI](https://circleci.com/gh/sonatype-nexus-community/nexus-repository-p2.svg?style=shield)](https://circleci.com/gh/sonatype-nexus-community/nexus-repository-p2) [![Join the chat at https://gitter.im/sonatype/nexus-developers](https://badges.gitter.im/sonatype/nexus-developers.svg)](https://gitter.im/sonatype/nexus-developers?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge) [![DepShield Badge](https://depshield.sonatype.org/badges/sonatype-nexus-community/nexus-repository-p2/depshield.svg)](https://depshield.github.io)
 
+> **Huzzah!** p2 is now part of Nexus Repository Manager. Version 3.20 includes the p2 plugin by default.
```
Comment (joedragons) [inline README.md:18]: Unless you're recording something else, I think p2 was 3.21 (ref https://help.sonatype.com/display/NXRM3/Release+Notes#ReleaseNotes-NexusRepositoryManager3.21.0)

Comment (proteus1121) [reply README.md]: Updated https://github.com/sonatype-nexus-community/nexus-repository-p2/pull/79/commits/badc6e7b769fdef8cc224cca99df41dcd02b6390

**Pull request merged**